builtin/logical/pki/path_acme_new_account.go

```diff
@@ -0,0 +1,159 @@
+package pki
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/vault/builtin/logical/pki/acme"
+	"github.com/hashicorp/vault/sdk/framework"
+	"github.com/hashicorp/vault/sdk/logical"
+)
+
+func pathAcmeRootNewAccount(b *backend) *framework.Path {
+	return patternAcmeNewAccount(b, "acme/directory")
+}
+
+func pathAcmeRoleNewAccount(b *backend) *framework.Path {
+	return patternAcmeNewAccount(b, "roles/"+framework.GenericNameRegex("role")+"/acme/directory")
+}
+
+func pathAcmeIssuerNewAccount(b *backend) *framework.Path {
+	return patternAcmeNewAccount(b, "issuer/"+framework.GenericNameRegex(issuerRefParam)+"/acme/directory")
+}
+
+func pathAcmeIssuerAndRoleNewAccount(b *backend) *framework.Path {
+	return patternAcmeNewAccount(b, "issuer/"+framework.GenericNameRegex(issuerRefParam)+"/roles/"+framework.GenericNameRegex("role")+"/acme/directory")
+}
+
+func patternAcmeNewAccount(b *backend, pattern string) *framework.Path {
+	return &framework.Path{
+		Pattern: pattern,
+		Fields:  map[string]*framework.FieldSchema{},
```

Comment:
Do we need to populate fields?

Reply:
Yeah, I forked this over before you added yours :D Whoops!